server/backend/api/app/schemas/privilege_details.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+from pydantic import BaseModel, ConfigDict, StringConstraints, model_validator
+from typing_extensions import Annotated, Literal
+
+from api.app.constants import (
+    CLIENT_NUMBER_MAX_LEN,
+    CLIENT_NAME_MAX_LEN,
+    DISTRICT_ID_MAX_LEN,
+    DISTRICT_NAME_MAX_LEN,
+    ROLE_NAME_MAX_LEN,
+)
+
+
+class ScopeSchema(BaseModel):
+    scope_type: Literal["Client", "District"]
+    client_id: Optional[
+        Annotated[str, StringConstraints(max_length=CLIENT_NUMBER_MAX_LEN)]
+    ] = None
+    client_name: Optional[
+        Annotated[str, StringConstraints(max_length=CLIENT_NAME_MAX_LEN)]
+    ] = None
+    district_id: Optional[
+        Annotated[str, StringConstraints(max_length=DISTRICT_ID_MAX_LEN)]
+    ] = None
+    district_name: Optional[
+        Annotated[str, StringConstraints(max_length=DISTRICT_NAME_MAX_LEN)]
+    ] = None
+
+
+class RoleSchema(BaseModel):
```

Comment:
the RoleSchema shouldn't show up as an import suggestion since it's not included in the __init__.py, to import this class, if i understand this correctly, devs will have to explicitly import this from the path like `from api.app.schemas.privilege_details import RoleSchema`, i guess this is an advantage of storing them in the same file. However, i do see problem with this name if we are going to use this schema somewhere else.

Anyway, i'll think of a better name

Reply:
renamed file name to [privilege_change_performer](https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/47df43ea3d5b21e9bd510c5761b9a2f82540cf72) and class name to [PrivilegeChangePerformerSchema](https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/4d17a314a9d8fd5e1d7fb6cc019714378f429ae5)